tests/CodeFileTestCase.php

```diff
@@ -45,16 +49,65 @@ public static function tearDownAfterClass(): void
      */
     private static function removeDirRecursive($dirName)
     {
+
         foreach (scandir($dirName) as $fileName) {
             if ($fileName !== '.' && $fileName !== '..') {
-                $filePath = "$dirName/$fileName";
+                $filePath = $dirName . DIRECTORY_SEPARATOR . $fileName;
                 if (is_dir($filePath)) {
                     static::removeDirRecursive($filePath);
                 } else {
                     unlink($filePath);
                 }
             }
         }
-        rmdir($dirName);
+
+        @rmdir($dirName);
+    }
+
+    /**
+     * Read file content ignoring whitespace characters
+     * @param $filename
+     * @return string
+     */
+    private function readFileContent($filename) {
+
+        $content = file_get_contents($filename);
+
+        if (!$content) {
+            return '';
+        }
+
+        return str_replace(["\t","\n","\r","\0","\x0B"],'', $content );
     }
+
+    /**
+     * Compare files by content
+     * @param $expectedFile
+     * @param $actualFile
+     * @return bool
+     */
+    public function assertFileEqualsIgnoreWhitespace($expectedFile, $actualFile ) {
```

Comment:
@actionm This method does not assert anything, it should not be named like an assertion. After stripping the white-spaces, I suggest Asserting that expected file is not empty, asserting that actual file is not empty, and then asserting that contents of both files are equal. This would make it an assertion method.
But then, I'm wondering why having an empty file (or a file with white spaces) would make this assertion fail?

Reply:
Also, I don't understand what's the purpose behind this. How does this contribute to the purpose of fixing tests on windows? Can you please explain